src/passes/ReorderLocals.cpp

```diff
@@ -54,10 +58,9 @@ struct ReorderLocals : public WalkerPass<PostWalker<ReorderLocals>> {
     // Gather information about local usages.
     walk(curr->body);
     // Use the information about local usages.
-    std::vector<Index> newToOld;
-    for (size_t i = 0; i < num; i++) {
-      newToOld.push_back(i);
-    }
+    std::vector<Index> newToOld((size_t)num);
+    // Fill newToOld from "0" to "num".
+    std::iota(newToOld.begin(), newToOld.end(), 0);
```

Comment:
This seems like an unrelated change. What is the advantage of using `std::iota` here?

Reply:
Using one line method similar to std::fill? But perhaps `std::iota` is quite exotic for many peoples. I could revert this